assets/startplexamp

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+FEATURES="UseSkiaRenderer"
+TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-tv.plex.Plexamp}"
+declare -a FLAGS=(--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --disable-gpu-blocklist)
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
```

Comment:
this whole thing is terrible because when you start from the DE (the desktop file is open), none of the console output is visible. If Wayland support is buggy, then it should be disabled so that things work out of the box.

Reply:
Thanks for the quick response! Wayland support is mostly rock solid, aside from some nvidia cards/drivers unfortunately. Although it's "hacky", this code is used by other Electron apps in Flathub, e.g. https://github.com/flathub/xyz.armcord.ArmCord as well as our existing Plexamp flatpak.

If it's a blocker for release, please let me know, but otherwise I'd prefer to ship Wayland support enabled for everybody with decent (read: not nvidia) graphics, and provide an escape hatch for anybody experiencing issues.

Reply:
it's not because another app does it that it's right. Even more when it was added back after the reviewers objected.

Reply:
I understand where you're coming from, but Wayland provides a much smoother experience than X11 for 99% of our Linux users, and for a tiny percentage of nvidia users (under specific distributions, running specific drivers) it presents problems. I'd rather provide a good experience to as many as possible, and provide an opt-out method for the tiny fraction of users than hold everyone back with a worse experience.

Until electron runs better for wayland + nvidia (with fixes either on the chromium end or the nvidia end), I don't see a better option than this, but I'm open to ideas.